api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,8 +111,27 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+		Date startDate = visit.getStartDatetime();
+		Patient patient = visit.getPatient();
+		Date birthDate;
+		if(patient!=null) {
+			birthDate = visit.getPatient().getBirthdate();
```

Comment:
Since you already have the patient variable, do you think you need to again call visit.getPatient()?